packages/config/src/index.ts

```diff
@@ -209,6 +230,7 @@ class TruffleConfig {
 
     config.merge(staticConfig);
     config.merge(options);
+    config.addDefaultNetworks();
```

Comment:
I think this should probably go in the constructor. Ugh! Unfortunately Truffle is not consistent about how it creates Truffle Config objects. Mostly I think it gets loaded using `Config.default()` or `Config.detect()`.

Reply:
So actually we might need it in `detect` as well.

Reply:
As far as `detect` goes, it calls `load`, so that case is already covered. But you're right that we should also include it in the default. Let me try moving it to the constructor.